sstables/storage.cc

```diff
@@ -86,7 +86,7 @@ class dir_wrapper final {
 // cannot define these classes in an anonymous namespace, as we need to
 // declare these storage classes as "friend" of class sstable
 class filesystem_storage final : public sstables::storage {
-    dir_wrapper _dir;
+    mutable dir_wrapper _dir;
```

Comment:
Suggest to call dir_wrapper opened_directory. It says something about the actual wrapping.

Reply:
Done in v5.

Reply:
... and documented in the v5 changelog.